### ISSUE TITLE
Step functions not resolved

### ISSUE THREAD

**Issue by ChrisLincoln** [TheBrainFamily/cypress-cucumber-preprocessor#418]
I am having issues bootstrapping a basic cypress-cucumber project. I've got a plain-jane Create-React-App project in the repo below that demonstrates my current state-of-the-art.

When I run my .feature from the cypress app, it appears that none of the When/Then functions are resolved.  My step functions are in a .ts module, named and located per the docs.  Here is the hello-world.feature:

`@id-68596b95-f04c-4ce9-b971-7d78acc66667
Feature: Hello World

  @id-0d0dcf31-8276-4599-aaff-3b549a5545a2
  Scenario: I visit the app for the first time.
    When I visit the app
    Then it says hello world.

`
Here is ./hello-world/hello-world.ts

`/// <reference types="cypress" />

import { When, Then } from "cypress-cucumber-preprocessor/steps";
When('I visit the app', () => {
  cy.visit('http://localhost:3001')
})
Then(/it says hello world./, () => {
  
})

`

### Test code to reproduce

This is a basic CRA app with cypress added.  I've got plain-old-cypress describe/it test written in TS working. See ordinary.spec.ts.ignored (rename to remove the .ignored to verify).

code for this repo is [here](https://github.com/ChrisLincoln/cypress-cucumber-ts-issue)

(Note:  the url for the cy.visit will probably be bogus for you. I've got other junk running.  I'd be happy if that line of code just executed.)

### Versions

* **Cypress version**: 4.11.0
* **Preprocessor version**: 2.5.4
* ** @cypress/browserify-preprocessor version: 3.0.1
* **Node version**: 12.16.2

**Comment by lgandecki** [comment]
Works for me.

run yarn
and then 
`./node_modules/.bin/cypress run` 

![image](https://user-images.githubusercontent.com/4002543/88147955-346e5880-cbfe-11ea-8c20-352978284d40.png)

**Comment by lgandecki** [comment]
Please clone the repo fresh and try to run just those steps, post specific error/screenshots if the error still happens :) Happy to reopen the issue then

**Comment by ChrisLincoln** [comment]
Cloned as requested, just for completeness.  I get the same results with cypress run.  It shows as passing when using the Cypress UI, too, but it didn't actually run anything from my TS file.  
![Screenshot from 2020-07-22 05-09-43](https://user-images.githubusercontent.com/16564511/88164533-e0517d00-cbd9-11ea-8cad-4ec65ca98a1d.png).

The "When()" in hello-world/hello-world.ts even has a 'cy.visity()' to my server, which isn't even running at the moment.
I woke up this morning with a theory, I'm gonna go off on that tangent. I'll report back

**Comment by ChrisLincoln** [comment]
My theory was all wet, but I did do some debugging and learned more.  I'm at a loss for discovering the magic of how this library marries up a When/Then/etc with the registered implementation for a particular feature. Everything looks common/global to me, but I can't see what the plugin's on(file:preprocessor) is up to. All I see is the final bundle. 

That curiosity led me back to this:  What is this 'cosmiconfig' stuff?  I have the feeling I shouldn't be glancing over that part.  I looked in the TS sample repo and didn't find evidence of it, so presumed it wasn't needed. That repo passes for me, but I noticed it was back on Cypress 3.  I'm going to dive back into it and see if I can break it just like mine is broken.

**Comment by lgandecki** [comment]
Thanks for the debugging.  Please keep updating here ,and I will jump in when possible.

Just one quick note. You actually do use the cosmiconfig, here:
https://github.com/ChrisLincoln/cypress-cucumber-ts-issue/blob/master/package.json#L42-L44

**Comment by ChrisLincoln** [comment]
I couldn't break your TS sample. So I painstakingly when through both apps line-by-line and noticed that the syntax highlighting was different in mine. Turns out I've got some non-space white-space in my .feature file after the When/Then which causes some subtle parsing problems in the library.  My files are generated from another app, which apparently has a bug that I get to go fix. I'd need to open the file in some other editor to figure out what I have wedged in there.

![Screenshot from 2020-07-22 10-13-57](https://user-images.githubusercontent.com/16564511/88194208-2ae7ef00-cc04-11ea-85a8-b2ceaa223021.png)
